README.md

```diff
@@ -1,161 +1,95 @@
+# The Official DocuSign C# Client
 
-Requirements
-============
+## Requirements
 
-Microsoft .NET version 4.5 or later.
+- Microsoft .NET version 4.5+ (4.6.1 recommended)
+- Free [Developer Sandbox](https://go.docusign.com/sandbox/productshot/?elqCampaignId=16531)
 
-Compatibility
-============
+## Compatibility
 
-* .NET Standard 2.0. See [.NET Standard Selector](http://immo.landwerth.net/netstandard-versions/#) for details on versioning for the individual components in .NET Standard.
-* .NET Core 2.0
-* .NET Framework 4.5 or later
-* TLS 1.2
+- .NET Standard 2.0. See [.NET Standard Selector](http://immo.landwerth.net/netstandard-versions/) for details on versioning for the individual components in .NET Standard
+- .NET Core 2.0
+- Microsoft .NET version 4.5+
+- TLS 1.2
 
-Installation
-============
+## Usage
 
-### NuGet Package Manager
+This open-source SDK is provided for cases where you would like to make additional changes that the SDK does not provide out-of-the-box. If you simply want to use the SDK with any of the examples shown in the [Developer Center](https://developers.docusign.com/esign-rest-api/code-examples), follow the installation instructions below.
 
-To add to a new or existing Visual Studio project:  
+## Installation
 
-	1. Open project and go to Tools -> NuGet Package Manager -> Manage NuGet Packages for Solution.
-	2. Search for "DocuSign", select the DocuSign.eSign package, and click Install.  
+### NuGet Package Manager:
 
-### Manual Install 
+1. Create a new C# project, or open an existing one.
+2. Click **Tools** -\&gt; **NuGet Package Manager** -\&gt; **Manage NuGet Packages for Solution**.
+3. Search for **DocuSign**.
+4. Select **DocuSign.eSign.dll**.
+5. Click the check box next to your project name.
+6. Click **Install**.
 
-Copy `DocuSign.eSign.dll` file to your local machine and add reference through your project settings.
+### Package Manager Console:
 
-### Others
+1. Open or create a new C# project.
+2. Open the **Package Manager Console** by either:
+  1. Clicking **Package Manager Console** along the bottom of Visual Studio
+  2. Clicking **Tools** -\&gt; **NuGet Package Manager** -\&gt; **Package Manager Console**
+3. In the **Package Manager Console** , type: **Install-Package DocuSign.eSign.dll**
 
-Alternatively you can just copy the source code directly into your project. 
+### Manual install:
 
-#### Dependencies
+Copy the _DocuSign.eSign.dll_ file to your local machine and add a reference to it through your project settings.
 
-This client has the following external dependencies: 
+## Dependencies
 
-* Newtonsoft.Json.dll
-* RestSharp.dll
-* BouncyCastle
-* System.IdentityModel.Tokens.Jwt
+This client has the following external dependencies:
 
-Usage
-=====
+### .NET Standard v2 and .NET Core:
 
-To send a signature request from a template using OAuth JWT authentication:
+- CSharp v4.5.0
```

Comment:
Should be Microsoft.CSharp

Reply:
Updated / alphabetized the Dependency list